### DISCUSSION TITLE
Select回显问题

### DISCUSSION THREAD

**Discussion by ChenKun1997** [ant-design/ant-design#44035] [Q&A]
场景是这样的，进入一个页面需要进行数据回显，但是此时Select框的Option也是需要用请求来获取的，但是此时我的value值已经拿到了，怎么进行回显呢，有没有最佳实践,本办法是需要先等获取Option的请求拿到数据再去获取需要回显的数据，但是这样请求会很慢

**Comment by linxianxi** [comment]
内部的逻辑是，找不到 label，就先显示 value。你可以用 Promise.all 并行请求
